config/serving.yaml

```diff
@@ -0,0 +1,41 @@
+# Full struct in cmd/prowgen/prowgen.go#Config
+
+config:
+  branches:
+    "release-v1.7":
+      cron: "0 17 * * 1-5"
+      openShiftVersions:
+        - 4.12
+        - 4.8
+    "release-v1.8":
+      cron: "0 19 * * 1-5"
+      openShiftVersions:
+        - 4.12
+        - 4.10
+    "release-v1.9":
+      cron: "0 21 * * 1-5"
+      openShiftVersions:
+        - 4.12
+        - 4.10
+    "release-next":
+      cron: "0 8 * * 1-5"
+      openShiftVersions:
+        - 4.12
+        - 4.10
+
+repositories:
+  - org: openshift-knative
+    repo: serving
+    imagePrefix: knative-serving
+    imageNameOverrides:
+      migrate: storage-version-migration
+    slackChannel: "#knative-serving-ci"
+    e2e:
+      matches:
+        - ".*e2e$"
+        - ".*e2e-tls$"
+    resources:
+      '*':
+        requests:
+          cpu: 500m
+          memory: 4Gi
```

Comment:
Is this actually needed? Like 4Gi? :exploding_head:

Reply:
I haven't checked, but all our builds are currently running with that. I would prefer to keep the current value to not introduce a potential flake while trying to get our builds running again with the new image tag pattern.